plugins/linear-genome-view/src/LinearGenomeView/components/OverviewRubberBand.tsx

```diff
@@ -164,10 +164,12 @@ function OverviewRubberBand({
           <Tooltip
             open={!mouseDragging}
             placement="top"
-            title={Math.max(
+            title={`${overview
+              .pxToBp(guideX)
+              .refName.toLocaleString()} ${Math.max(
```

Comment:
refName probably doesn't need toLocaleString

Reply:
(toLocaleString is only needed for numbers, it adds commas to long numbers, etc. Note that any use of this should use value.toLocaleString('en-US'))

That is using the US system of comma-fying long numbers which is US biased but it's just how the rest of the app displays numbers now